.vscode/launch.json

```diff
@@ -12,8 +12,8 @@
       "port": 1044,
       "timeout": 30000,
       "sourcePaths": [
-        "${workspaceRoot}/com.microsoft.java.debug.core/main/java",
-        "${workspaceRoot}/com.microsoft.java.debug.plugin/src/main/java"
```

Comment:
The `workspaceRoot` parameter is deprecated in vscode. `workspaceFolder` is now recommended